docs/sources/enterprise/usage-insights.md

```diff
@@ -55,3 +55,16 @@ It shows two kinds of information:
 In the search view, you can sort dashboards using these insights data. It helps you find unused or broken dashboards or discover most viewed ones.
 
 {{< docs-imagebox img="/img/docs/enterprise/improved_search.png" max-width="650px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+Data source insights give you information about how much of the data source has been used and how many errors it has returned. It shows the last thirty days of data.
```

Comment:
```suggestion
Data source insights give you access to the following information for the last thirty days:
- Queries per day
- Errors per day
- Average load duration per day (ms)

```

Reply:
I am OK either way. However, I prefer the former as it introduces what the insights feature does.

Reply:
I changed it around a little, trying to get the good parts of both ways of writing the docs.